web/src/cypress/e2e/escalationPolicySteps.cy.ts

```diff
@@ -202,7 +202,6 @@ function testSteps(): void {
             .should('contain', s1.delayMinutes)
             .parent('[tabindex]')
             .focus()
-            .type(' ')
```

Comment:
I think this is significant because the spacebar is used to lift/drop items, maybe add a `cy.type` after

Reply:
Yeah, that was causing the test to fail. Should be resolved now.